app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -161,8 +161,12 @@ def verify_virt_supported(opts)
     virt_supported
   end
 
+  def authentication_status(type = default_authentication_type)
+    authentication_best_fit(type).try(:status)
+  end
+
   def authentication_status_ok?(type = default_authentication_type)
-    authentication_best_fit(type).try(:status) == "Valid"
+    authentication_status == "Valid"
```

Comment:
I think we need to pass `type` in here in the event that someone passes a non-default type to this method
```suggestion
    authentication_status(type) == "Valid"
```